webapp/src/DOMJudgeBundle/Controller/Jury/BalloonController.php

```diff
@@ -184,19 +180,35 @@ public function indexAction(Request $request, Packages $assetPackage, KernelInte
                     ])]];
             }
 
+        
             $balloons_table[] = [
                 'data' => $balloondata,
                 'actions' => $balloonactions,
                 'cssclass' => $balloon->getDone() ? 'disabled' : null,
             ];
         }
 
+        // Load preselected filters
+        $filters              = $this->dj->jsonDecode((string)$this->dj->getCookie('domjudge_balloonsfilter') ?: '[]');
+        $filteredAffiliations = [];
+        if (isset($filters['affiliation-id'])) {
+            /** @var TeamAffiliation[] $filteredAffiliations */
+            $filteredAffiliations = $this->em->createQueryBuilder()
+                ->from('DOMJudgeBundle:TeamAffiliation', 'a')
+                ->select('a')
+                ->where('a.affilid IN (:affilIds)')
+                ->setParameter(':affilIds', $filters['affiliation-id'])
+                ->getQuery()
+                ->getResult();
+        }
+
         return $this->render('@DOMJudge/jury/balloons.html.twig', [
             'refresh' => ['after' => 60, 'url' => $this->generateUrl('jury_balloons')],
```

Comment:
This now needs to also have `'ajax' => true` to let the page refresh using AJAX.